### PR TITLE
Update reclaim policy documentation to be consistent with field names

### PR DESCRIPTION
**Pull request by msau42**
* Document the actual field values for reclaim policy
* Move delete reclaim policy up since that is more common

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/website/7487)

### CODE REVIEW

**Comment by msau42** [comment]
Partially fixes #7469

**Comment by saad-ali** [comment]
LGTM

File: docs/concepts/storage/persistent-volumes.md [hunk 1]
```diff
@@ -98,20 +98,24 @@ Finalizers:    [kubernetes.io/pvc-protection]
...
 1. Manually clean up the data on the associated storage asset accordingly.
 1. Manually delete the associated storage asset, or if you want to reuse the same storage asset, create a new `PersistentVolume` with the storage asset definition.
 
-#### Recycling
+#### Delete
 
-**Warning:** The recycling reclaim policy is being deprecated. Instead, the recommended approach is to use dynamic provisioning.
+For volume plugins that support the `Delete` reclaim policy, deletion removes both the `PersistentVolume` object from Kubernetes, as well as deleting the associated storage asset in the external infrastructure, such as an AWS EBS, GCE PD, Azure Disk, or Cinder volume. Volumes that were dynamically provisioned inherit the [reclaim policy of their `StorageClass`](#reclaim-policy), which defaults to `Delete`. The administrator should configure the `StorageClass` according to users' expectations, otherwise the PV must be edited or patched after it is created. See [Change the Reclaim Policy of a PersistentVolume](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/).
```
Comment (tengqm) [inline docs/concepts/storage/persistent-volumes.md]: > as well as deleting the associated storage

as well as the associated storage

Comment (tengqm) [inline docs/concepts/storage/persistent-volumes.md]: >https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/

please use relative path, i.e. /docs/tasks/...

**Comment by msau42** [comment]
Updated

**Pull request merged**